charts/trustify/templates/helpers/_opentelemetry.tpl

```diff
@@ -0,0 +1,16 @@
+{{/*
+Shares the OTEL_EXPORTER_OTLP_ENDPOINT env var with tracing and metrics signals
+
+NOTE: Using the `tracing.collector` as the flag to switch between 'custom' and
+'default' OTEL collector endpoint, as both back-ends (Jaeger and Prometheus)
+are making use of it.
+
+
+Arguments (dict):
+  * root - .
+  * module - module object
+*/}}
+{{ define "trustification.application.tracing.collector"}}
+- name: OTEL_EXPORTER_OTLP_ENDPOINT
+  value: {{ if .root.Values.tracing.collector | default "" | trim | eq "" }}"http://infrastructure-otelcol:4317"{{ else }}{{ .root.Values.tracing.collector | quote }}{{ end }}
```

Comment:
could be put this under `.opentelemetry` instead?

Reply:
It worked, PR updated thanks for the review and suggestion now it is also more clean than before :+1: